choco/nunit-console-with-extensions.nuspec

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-console-with-extensions</id>
+    <version>0.0.0</version>
```

Comment:
I see it gets injected in the Cake file, just thought it should be `$version$` like NuGet does it. I could very easily be wrong though 😄